NeoML/test/src/DnnSolverTest.cpp

```diff
@@ -813,3 +829,64 @@ TEST( CDnnLambGradientSolverTest, Serialization4 )
 	lamb->SetWeightDecayClip( 2.5f );
 	solverSerializationTestImpl( lamb.Ptr(), true );
 }
+
+// ====================================================================================================================
+
+static void checkBlobInequality( CDnnBlob& firstBlob, CDnnBlob& secondBlob )
+{
+	ASSERT_TRUE( firstBlob.HasEqualDimensions( &secondBlob ) );
+	const int dataSize = firstBlob.GetDataSize();
+	float* first = firstBlob.GetBuffer<float>( 0, dataSize, true );
+	float* second = secondBlob.GetBuffer<float>( 0, dataSize, true );
+	bool success = false;
+	for( int i = 0; i < dataSize; ++i ) {
+		success |= !FloatEqImpl( first[i], second[i], 1e-4f );
```

Comment:
After the first `true` you may skip rest checks.

Reply:
It will be removed #869